tests/parsers/xyz.js

```diff
@@ -12,3 +12,14 @@ describe("Parsers:XYZ", () => {
         ]);
     });
 });
+
+describe("Parsers:XYZ", () => {
```

Comment:
@timurbazhirov is this sufficient?

Reply:
<img src="https://media4.giphy.com/media/tIeCLkB8geYtW/giphy.gif"/>